### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DerrickXuNu**
Documentation and License finalization.

### CODE REVIEW

File: docs/md_files/contributor.md [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
+=======
 - Help on Sumo part
 - [Linkedin](https://www.linkedin.com/in/yi-guo-4008baaa/)
 
-### Supervisor
-Dr. Jiaqi Ma (Associate Professor @ UCLA)
-- Project and lab supervisor
-- [Linkedin](https://www.linkedin.com/in/jiaqi-ma-17037838/)
-- [Google Scholar](https://scholar.google.com/citations?user=S3cQz1AAAAAJ&hl=en)
+>>>>>>> 232a18fe1724891b03327ab54f14f5849ae11214
```
Comment (XHwind) [inline docs/md_files/contributor.md:43]: Seems that the merge conflicts are not resolved here.  (like <<<<<HEAD)

File: docs/md_files/introduction.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
 ## OpenCDA Overview
 Current autonomous driving simulation platforms that support scene rendering and traffic simulation mainly concentrate
-on single-vehicle intelligence, therefore, developing and testing Cooperative Driving Automation applications(e.g., cooperative perception,
+<<<<<<< HEAD
```
Comment (XHwind) [inline docs/md_files/introduction.md:3]: Seems that merge conflicts are not resolved here~

File: docs/md_files/contributor.md (hunk 1 above)
Comment (DerrickXuNu) [reply docs/md_files/contributor.md]: yes, i fixed it

**Review by XHwind — CHANGES_REQUESTED** [review]
Please resolve some merge conflicts before merging~

**Review by XHwind — APPROVED** [review]

**Pull request merged**